src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/Validation/ModelValidatorProviderContext.cs

```diff
@@ -39,11 +40,11 @@ public IReadOnlyList<object> ValidatorMetadata
         }
 
         /// <summary>
-        /// Gets the list of <see cref="IModelValidator"/> instances. <see cref="IModelValidatorProvider"/> instances
+        /// Gets the list of <see cref="ValidatorItem"/> instances. <see cref="IModelValidatorProvider"/> instances
         /// should add validators to this list when
```

Comment:
I'd change this comment to say `should set the appropriate ValidatorItem.Validator properties when called`.